footer.php

```diff
@@ -282,6 +283,10 @@ function make_navbutton($imgsrc, $bgcolour, $title, $url, $newwindow = false) {
         }
         $output .= 'margin-right: 5px;" width="50" height="50" /></a>';
     } else {
+		$formClass = 'navbuttontext';
```

Comment:
Please don't use camelcase for variables - $formclass is the correct name, according to the Moodle coding guidelines.